pacoloco.go

```diff
@@ -78,10 +106,79 @@ var (
 	downloadingFilesMutex sync.Mutex
 )
 
+// force resources prefetching
+func prefetchRequest(url string) (err error) {
+
```

Comment:
the existing code does not add an empty line after the function name. Let's be consistent with the rest of the code.